### ISSUE TITLE
Vimeo iframe does not pass events to parent

### ISSUE THREAD

**Issue by pgouldmiro** [vimeo/player.js#953]
We currently use the Vimeo iframe within a modal. 

We listen for keyboard events, in particular the `Esc` key to close the modal. This is important both from a usability perspective, but as well as accessibility to ensure keyboard-only users are not trapped in content ([WCAG Success Criterion 2.1.2](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/keyboard-operation-trapping.html)).

It appears that no user input events get passed from the Vimeo iframe back to the parent. This means that if the user is focused within the Vimeo iframe, we cannot intercept the `Esc` key event to close the modal. 

### Expected Behavior
User input event should be passed back to the parent, preferably using `window.postMessage()`.

### Actual Behavior
No user input events get passed from the Vimeo iframe back to the parent

### Steps to Reproduce
1. Embed a Vimeo iframe onto a page
2. Using the `Tab` key, tab until one of the elements within the Vimeo iframe is focused
3. Press the `Esc` key
4. Event listeners on the parent page will not receive the event

**Comment by rkrishnan8594** [comment]
Thanks for leaving this feedback @pgouldmiro. This is not a feature the SDK currently supports, but we've filed an internal ticket to explore eventually adding support.

**Comment by Benzo-Fury** [comment]
3 years later and still nothing 🥲. Bump 👍.

The Iframe should be able to forward keypresses from inside the Iframe back to the caller. Currently, developers need to choose between keybinds only working when the iframe is in focus, or keybinds only working when the iframe is NOT in focus. Both destroying proper UX principles. 

All solutions around this that I've found seem to be hacky and temperamental.
